modules/end-user-guide/examples/snip_che-editor-key-values.adoc

```diff
@@ -0,0 +1,26 @@
+.The `__<editor_key>__` values for the supported IDEs
+|===
+| IDE | `__<editor_key>__` value | Note
+
+
+| link:https://github.com/eclipse-che/che-theia[Che-Theia]
+| `eclipse/che-theia/latest`
+| This is the default IDE that loads in a new workspace without this URL parameter.
```

Comment:
```suggestion
| The default IDE that loads in a new workspace if you do not specify [which URL parameter?].
```

Reply:
- Consider making the structure of all these definitions parallel. You can use a noun-based fragment, like I've suggested here. Or make them full sentences, like you've done in the next one. 
- In "this URL parameter," I'm not sure what the antecedent of "this" is. Which URL parameter does it refer to?